README.md

```diff
@@ -1 +1,69 @@
-# EzSystemsPrivacyCookieBundle
+# Privacy Cookie Bundle
+
+This bundle adds privacy cookie banner into eZ Publish / eZ Platform.
+
+## Requirements
+
+- eZPublish 5.4/2014.11 or above
+- BootstrapJS v3.x
```

Comment:
This is not mentioned in composer requirements, should it?
Not hard coded to a specfic version here, rather 3.x if that is possible for instance to allow wide reuse of the bundle.